TestResultSummaryService/DataManager.js

```diff
@@ -62,8 +62,12 @@ class DataManager {
 
     async updateBuildWithOutput( data ) {
         logger.verbose( "updateBuildWithOutput", data.buildName, data.buildNum );
-        const { _id, buildName, output, ...newData } = data;
+        let { _id, buildName, output, ...newData } = data;
         const criteria = { _id: new ObjectID( _id ) };
+        const size = 40 * 1024 * 1024;

```

Comment:
Could we add comments? We temporarily limit the output size to be 40M until test parser logic is updated  https://github.com/AdoptOpenJDK/openjdk-test-tools/issues/65